app/views/reports/task_report.html.erb

```diff
@@ -0,0 +1,74 @@
+
+<div class="container datatable-container">
+  <div class="row">
+      <div class="col">
+        <div class="row">
+          <div class="col-md-4">
+            <b>Project:</b>
+          </div>
+          <div class="col-md-8">
+            <input id="project" type="text" class="form-control" />
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="row">
+          <div class="col-md-4">
+            <b>Status:</b>
+          </div>
+          <div class="col-md-8">
+            <select id="status" class="form-control">
+              <option value="">--Select Status--</option>
```

Comment:
could use ruby code ( a loop ) to create all inputs in select

Reply:
something like this
<% for i in %w('new in-progress ..') %>
<option> <%= i %> </option>